scriptmodules/helpers.sh

```diff
@@ -115,6 +115,7 @@ function addLineToFile() {
     if [[ -f "$2" ]]; then
         cp -p "$2" "$2.bak"
     else
+        touch "$2"
         sed -i --follow-symlinks '$a\' "$2"
```

Comment:
I think I may have introduced an error in this function as the sed is only being run if the file doesn't exist - which is wrong. It should be run if it does exist  (to make sure it ends with a LF).

Reply:
I can fix it after the PR anyway - the fix should be done in a separate commit from this function addition etc.